app/views/tweets/index.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" type="text/css" href="/style.css">
+    </head>
+    <body>
+    <div class="body">
+      <div class="header">
+        <h1>Welcome, <%= @user.username %></h1>
+      </div>
+      <div class="banner">
+        <h2>Tweets</h2>
+      </div>
+      <div class ="tweet_background">
+        <div class="tweets">
+          <% Tweet.all.each do |tweet| %>
```

Comment:
It's not really within the MVC framework to make a database call (`Tweet.all` is a database call, try it in rails console and you'll see the SQL query!) within the view. We should be loading up this data in the controller.